### DISCUSSION TITLE
Posts about Granian 1.0.0

### DISCUSSION THREAD

**Discussion by tecbr** [emmett-framework/granian#183] [General]
What do you think about posting about the Granian 1.0.0 release on Python-related social media?

I think the author should post it cause he can answer any questions.

Reddit: [Python](https://www.reddit.com/r/python/), [Django](https://www.reddit.com/r/django/), [Flask](https://www.reddit.com/r/flask/), [FastAPI](https://www.reddit.com/r/fastapi/)...
HackerNews...

**Comment by gi0baro** [comment]
Posted [on Reddit](https://www.reddit.com/r/Python/comments/19cs4qx/granian_10_is_out/) and [on HN](https://news.ycombinator.com/item?id=39087919)

**Comment by tecbr** [comment]
Posted on /r/Django, /r/Flask and /r/FastAPI (Waiting for moderation to approve)

I think you already know cause I saw you there =)
